client/api_object.go

```diff
@@ -474,7 +475,45 @@ func (c *client) ListObjects(ctx context.Context, bucketName string, opts types.
 		return types.ListObjectsResult{}, err
 	}
 
+	if opts.MaxKeys == 0 {
```

Comment:
we should set the defaut maxKeys value here.   It is allowed not pass the values and be 0.

`var maxKey uint64
if opts.MaxKeys == 0 {
      opts.MaxKeys = default.value
} 
`